<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Template_Image_Preview_View</class>
 <widget class="QWidget" name="Template_Image_Preview_View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>1920</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="preview_template_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>150</y>
     <width>1030</width>
     <height>1030</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="raw_image_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>1210</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(149, 149, 149);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_with_background_label">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>1210</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(149, 149, 149);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_capture_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>1740</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>CAPTURE AGAIN</string>
   </property>
   <property name="icon">
    <iconset resource="UI_ViewResource.qrc">
     <normaloff>:/View/Icon/restart_icon.png</normaloff>:/View/Icon/restart_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_capture_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>1740</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="UI_ViewResource.qrc">
     <normaloff>:/View/Icon/check_icon.png</normaloff>:/View/Icon/check_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="export_template_button">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>1740</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>EXPORT TEMPLATE</string>
   </property>
   <property name="icon">
    <iconset resource="UI_ViewResource.qrc">
     <normaloff>:/View/Icon/export_template_icon.png</normaloff>:/View/Icon/export_template_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="image_info_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>1030</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>IMAGE #...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="preview_countdown_label">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>25</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>CONTINUE IN ...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="UI_ViewResource.qrc"/>
 </resources>
 <connections/>
</ui>
